### ISSUE TITLE
"Spec" Options on Website, when booking VIP's

### ISSUE THREAD

**Issue by DruidEmmaNielsen** [Icecrown-Community/ICC-Member-Website-Public#120]
Wanted Change:  an additional booking field called "VIP Specialization" with the 4x options of "Melee, Healer, Tank, Ranged DPS" Preferably "must be filled" if VIP count is more than 0. 

The Issue: Each In-House Heroic Raids especially, and future In-House Mythic raids probably. There is a Discussion phase ONCE all Boosters + Clients joined raid. which means we are nearing our official raid start timer. This Discussion phase includes whispering all Clients their preferred VIP's.  Some might instantly reply, some might be on a bio break, or smoking. etc.  The Raid Leader doesn't know if they want Tank, Heal or Dps Loot.  All Boosters wants to know what to put their Loot Spec on.   So we end up with a  Discussion phase of 5-10mins  Every Time. 

Solution:  There are 2x ways this can be solved.  Every advertiser who books a VIP client, could type in Note their clients preferred Loot Spec.  Or...   We could have a Quality of Life improvement on the website where it would make things look Neat so that we do not have to get the Raid Leader read every single Note. with this new information Raid Leaders pre-hand could prepare a RT Note where we usually can see our Clients and VIP list, but moving forward, also see their Loot Specialization as well.   Saving us 4-9mins time  Every single In-House runs.

Personal Comment:  The reason I type 4-9mins is that after several heroic nights, it has become clear that Clients easily run's off. The sec they accept a summon, they realize "oh I should quick go bio break,  get a smoke, or perhaps have some snack for this ride"  preparing themselves for the next 1½ - 2 hours boost run.   So in this moment, trying to whisper them in my honest opinion, kind of looks a bit whack, and the entire Ted Talk phase could be entirely skipped.

**Comment by Dejnaa** [comment]
As a quick-fix solution we can ask all Advertisers to write in the note:
- Specc requested
- Item requested
- If they want different speccs on different bosses.

Yesterdays raid I left this note after talking to raidlead:

"4x leather VIPs, 1 Tank Trinket VIP"
The reason I didnt write 4x Boomkin VIPs is cause there was none left, so I could take any leather class :)
I also told the raidlead that its okey if the 5th was not leather as long as it could get the trinket. 

I could have detailed it better in the note, with yesterdays example would it have been better with:
"4x Intellect DPS Leather Users, 1x Tank VIP of any armortype for Trinket drop on SunKing - Intellect DPS specc for rings/neck/cape if possible on the other bosses."

Is that sufficient information? 
I would like to announce it with a template for everyone to use, so GCs/RLs dont spend 30min trying to read notes instead XD

**Comment by Dejnaa** [comment]
@DruidEmmaNielsen did you see above?

**Comment by DruidEmmaNielsen** [comment]
> @DruidEmmaNielsen did you see above?
Yes, read your reply the day you wrote it, but it says "we can ask"  and isn't a "we will ask" yet (:

**Comment by iElsha** [comment]
@Bgabor997 how much hour can it take?

**Comment by Bgabor997** [comment]
hmmm, so we just want to select it from "Melee, Healer, Tank, Ranged DPS", or we want an actuall spec select, what is almost done in another feature? bcs implementing that component for raid bookings, wouldn't take more than a hour or 2 i think

**Comment by Dejnaa** [comment]
We currently have this in PVP booking section, we would really just have to press what specc the person WANT loot for. 
If they want different on different bosses that would need to be noted in NOTE anyway :)

@bgabor997

**Comment by Bgabor997** [comment]
yea, since we have the component already it would take max 1 hour **with tests**

**Comment by Dejnaa** [comment]
@iElsha We await your approval Warchief

**Comment by iElsha** [comment]
Approving what?

**Comment by Dejnaa** [comment]
@iElsha Approving of Gabor putting 1h (inc tests) into getting this done and shipped.

**Comment by iElsha** [comment]
That's more than one hours and Gabor's mostly free of handling websites requests based on priority - it's a SEV4 we mostly want SEV 1 -> SEV 2 -> SEV 3 to be handled before.
If this is a high priority and the severity is wrong we can increase/decrease it

**Comment by Bgabor997** [comment]
@DruidEmmaNielsen we sill need this? Bcs we implemented the new VIP system.
